Gemfile.lock

```diff
@@ -379,7 +379,7 @@ GEM
       sprockets (> 3.0)
       sprockets-rails
       tilt
-    sexp_processor (4.13.0)
+    sexp_processor (4.14.1)
```

Comment:
### [sexp_processor](https://github.com/seattlerb/sexp_processor)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 4.13.0 → 4.14.1

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **9** commits. See the full changes on [the compare page](https://github.com/seattlerb/sexp_processor/compare/v4.13.0...v4.14.1).

These are the individual commits:
- (52db357) [Removed old maglev skips in test](https://github.com/seattlerb/sexp_processor/commit/52db35748c18e1a3ced84950cd5e54be07acc7b7)
- (5987c8d) [+ Added Klass matcher to match on types.](https://github.com/seattlerb/sexp_processor/commit/5987c8dc0452b7444c6872e5e2f410209d2dc3a2)
- (d43ca07) [+ Added '-' as an alias for the 'not?' pattern matching comm…](https://github.com/seattlerb/sexp_processor/commit/d43ca0703e246c1a51ba44eddecc6bced7d5b655)
- (17621e8) [+ Added any matcher to pattern parser.](https://github.com/seattlerb/sexp_processor/commit/17621e82596c886bba70054d18cc05fdb8c63962)
- (4660187) [prepped for release](https://github.com/seattlerb/sexp_processor/commit/4660187826856e1370f0936b6f15948254e54f12)
- (91d29e8) [Added tests for uppercase symbols staying symbols](https://github.com/seattlerb/sexp_processor/commit/91d29e8724aef3a39ed2f2770ae072465a947bd2)
- (ff18275) [- Fixed alias for `not?` to `-` for ruby <= 2.4. (nard-tech)…](https://github.com/seattlerb/sexp_processor/commit/ff18275137310afac99c1eeeb2f455507374eb39)
- (23a1777) [- Declared support for ruby 2.2+ to gemspec.](https://github.com/seattlerb/sexp_processor/commit/23a17771a12786d280039d93d05bd0de629a946a)
- (e59f67a) [prepped for release](https://github.com/seattlerb/sexp_processor/commit/e59f67aa54028c6c903c6720b71cc3cad94a4224)


</details>